server/middleware/setUpWebSecurity.ts

```diff
@@ -25,7 +25,7 @@ export default function setUpWebSecurity(): Router {
           // This ensures only scripts we trust are loaded, and not anything injected into the
           // page by an attacker.
           scriptSrc: ["'self'", (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`],
-          styleSrc: ["'self'", (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`],
+          styleSrc: ["'self'", "'unsafe-inline'"],
```

Comment:
Is there any way we can avoid doing this? Using 'unsafe-inline' opens the door a bit more for CSS injection

Reply:
Updated